packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -172,6 +192,16 @@ async function executeCompatibilityCheck(templateDir) {
               'Tutorial - Prerequisites https://redwoodjs.com/docs/tutorial/chapter1/prerequisites',
           }
         )}`,
+        '',
+        `  As an alternative solution, you can launch a Redwood project using GitPod instead. GitPod is a an online IDE.`,
+        `  See: ${terminalLink(
+          'Launch Redwood using GitPod',
+          'https://gitpod.io/#https://github.com/redwoodjs/starter',
+          {
+            fallback: () =>
+              'Launch Redwood using GitPod https://gitpod.io/#https://github.com/redwoodjs/starter',
+          }
+        )}`,
```

Comment:
This, and other texts are repeated three times here. What do you think about extracting it out into a function or variable that you can reuse instead of copy and pasting it?

Reply:
Duh! ... Of course! Excellent feedback. Thank you!